frontend/src/pages/HomePage.tsx

```diff
@@ -1,9 +1,83 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useState, useEffect } from 'react';
 import ApartmentCard from '../components/Home/ApartmentCard';
-import { Box, Container, Grid, Typography } from '@material-ui/core';
+import { Box, Container, Grid, Typography, Link } from '@material-ui/core';
 import Autocomplete from '../components/Home/Autocomplete';
+import { Apartment, LandlordWithId, Review } from '../../../common/types/db-types';
+import { Link as RouterLink } from 'react-router-dom';
+import get from '../utils/get';
 
 const HomePage = (): ReactElement => {
+  const [homeData, setHomedata] = useState<any>([]);
+  const [buildingData, setBuildingData] = useState<Apartment[]>([]);
+  const [landlordData, setLandlordData] = useState<LandlordWithId[]>([]);
+  const [reviewsBuilding1, setReviewsBuilding1] = useState<Review[]>([]);
```

Comment:
We probably don't need a state for each of these buildings: you can simply make one type of `useState<[Review, Review, Review]>` which is basically a tuple in TypeScript